influxdb/content.md

```diff
@@ -1,643 +1,262 @@
-
-Using the `docker run` [`-P, --publish-all` flag](https://docs.docker.com/reference/cli/docker/container/run/#publish-all) exposes the InfluxDB HTTP API to the host.
+This command maps port `8086` and mounts your current directory to persist data.
 
-## Configure InfluxDB v1 in a container
+For more information, see the [InfluxDB v1 Docker documentation](https://docs.influxdata.com/influxdb/v1/introduction/install/docker/).
 
-To configure InfluxDB v1 in a container, use a configuration file or environment variables.
+## How to use the InfluxDB Enterprise v1 Docker image
 
-### Use a configuration file
+*InfluxDB Enterprise v1 is an earlier version of InfluxDB Enterprise. InfluxDB 3 Enterprise is the latest stable version.*
 
-To customize and mount a configuration file, do the following:
+Use the official `influxdb:meta` and `influxdb:data` Docker images to deploy and manage an InfluxDB v1 Enterprise cluster. A valid license is required.
 
-1.	Output the current server configuration to a file in the mounted configuration directory--for example:
+### Start InfluxDB v1 Enterprise Cluster
 
-	```bash
-	docker run --rm %%IMAGE%%:1.8 influxd config > influxdb.conf
-	```
-
-2.	Edit the `influxdb.conf` file to customize [server configuration options](https://docs.influxdata.com/influxdb/v2/reference/config-options/#configuration-options).
-
-	```bash
-	docker run -p 8086:8086 \
-	  -v "$PWD/influxdb.conf:/etc/influxdb/influxdb.conf:ro" \
-	  %%IMAGE%%:1.8 -config /etc/influxdb/influxdb.conf
-	```
-
-	Replace `$PWD` with the host directory where you want to store the configuration file.
-
-### Use environment variables
-
-Pass [`INFLUXDB_` environment variables](https://docs.influxdata.com/influxdb/v1/administration/config/#environment-variables) to override specific InfluxDB v1 configuration options. An environment variable overrides the equivalent option in the configuration file.
+Create a custom Docker network for node communication:
 
 ```bash
-docker run -p 8086:8086 \
-    -e INFLUXDB_REPORTING_DISABLED=true \
-    -e INFLUXDB_META_DIR=/path/to/metadir \
-    -e INFLUXDB_DATA_QUERY_LOG_ENABLED=false \
-    %%IMAGE%%:1.8
+docker network create influxdb
 ```
 
-Learn more about [configuring InfluxDB v1](https://docs.influxdata.com/influxdb/v1.8/administration/config/).
-
-## Graphite
-
-InfluxDB supports the Graphite line protocol, but the service and ports are not exposed by default. To run InfluxDB with Graphite support enabled, you can either use a configuration file or set the appropriate environment variables. Run InfluxDB with the default Graphite configuration:
+Start three meta nodes (each with a unique hostname and license key):
 
 ```bash
-docker run -p 8086:8086 -p 2003:2003 \
-    -e INFLUXDB_GRAPHITE_ENABLED=true \
-    %%IMAGE%%:1.8
+# Meta node 0
+docker run -d \
+  --name=influxdb-meta-0 \
+  --network=influxdb \
+  -h influxdb-meta-0 \
+  -e INFLUXDB_ENTERPRISE_LICENSE_KEY=your-license-key \
+  influxdb:meta
```

Comment:
These generic aliases are no longer maintained, right?  So this should probably be `1.11-meta` instead, if the content is kept here?